pyramid/scaffolds/alchemy/development.ini_tmpl

```diff
@@ -26,7 +26,7 @@ sqlalchemy.url = sqlite:///%(here)s/{{project}}.sqlite
 
 [server:main]
 use = egg:waitress#main
-listen = 127.0.0.1:6543 [::1]:6543
+listen = localhost:6543
```

Comment:
Deprecated does not mean we reject changes to things. User contributed changes are always welcome for review but are generally subject to more scrutiny. Also it usually discourages people from touching the feature but if the work is done then I'm happy taking it.